app/src/main/java/com/github/palFinderTeam/palfinder/meetups/fragments/CriterionsFragment.kt

```diff
@@ -11,41 +10,48 @@ import androidx.fragment.app.DialogFragment
 import com.github.palFinderTeam.palfinder.R
 import com.github.palFinderTeam.palfinder.meetups.activities.MeetUpCreationViewModel
 import com.github.palFinderTeam.palfinder.utils.CriterionGender
-import org.florescu.android.rangeseekbar.RangeSeekBar
-import org.w3c.dom.Text
+import com.google.android.material.slider.RangeSlider
 
+const val MIN_AGE_DIST = 1.0f

```

Comment:
maybe create a companion object to store it, so it is usable outside of this class